test/rulesets/Base/base.jl

```diff
@@ -154,7 +165,7 @@
         end
     end
 
-    @testset "trinary ($f)" for f in (muladd, fma)
+    @testset "trinary ($f)" for f in (muladd, fma, clamp)
```

Comment:
This testset is not surficient for testing `clamp` as it has modal behavour.
Need to explictly test the 3  cases